src/main/java/de/tum/in/www1/artemis/web/rest/repository/RepositoryProgrammingExerciseParticipationResource.java

```diff
@@ -170,6 +170,12 @@ boolean canAccessRepository(Long participationId) throws IllegalArgumentExceptio
         return super.getFile(participationId, filename);
     }
 
+    @Override
+    @GetMapping(value = "/repository/{participationId}/fileType", produces = MediaType.TEXT_PLAIN_VALUE)
```

Comment:
you can specify collection name for the resource: 'repository/participations/participationId/filetype

Reply:
I removed this endpoint alltogether